src/ConvertInputFormat/main.cpp

```diff
@@ -275,14 +273,14 @@ bool processInput(
         auto const input_file = idf_parser->encode(epJSON, schema);
         std::string convertedEpJSON(outputDirectory + inputFileNameOnly + ".idf");
         EnergyPlus::FileSystem::makeNativePath(convertedEpJSON);
-        std::ofstream convertedFS(convertedEpJSON, std::ofstream::out | std::ofstream::binary);
+        std::ofstream convertedFS(convertedEpJSON, std::ofstream::out);
```

Comment:
Remove `binary` option for epJSON and idf output so they have dos line endings on windows. Note this matches [similar lines](https://github.com/NREL/EnergyPlus/blob/develop/src/EnergyPlus/InputProcessing/InputProcessor.cc#L303) in the main EnergyPlus code.

Reply:
This is fine. I likely just copied and pasted for all the various `ofstream` since cbor, msgpack, ubjson, and bson all require the binary flag since they are binary formats.